src/Raven.Studio/wwwroot/Content/css/pages/client-configuration.less

```diff
@@ -0,0 +1,14 @@
+#client-configuration-view {
+
```

Comment:
why we need that?

Reply:
just because I used it in the _less_ file

Reply:
?